src/resources/parser/glb-container-resource.js

```diff
@@ -190,6 +192,45 @@ class GlbContainerResource {
         return GlbContainerResource.createSceneHierarchy(sceneClones, 'Entity');
     }
 
+    // get material variants
+    getMaterialVariants() {
+        return Object.keys(this.data.variants);
+    }
+
+    // apply material variant to entity
+    applyMaterialVariant(name, entity) {
+        const variant = this.data.variants[name];
+        if (variant === undefined) {
```

Comment:
it could be just `if (!variant)` as you had before. It's a string, so that should be ok. It's only a problem if it is a number and can be 0. Same below.

Reply:
It can be 0 :)

Reply:
ah right, this is the index!

Reply:
Yep!